src/Compilers/CSharp/Test/Syntax/Parsing/StatementParsingTests.cs

```diff
@@ -2597,7 +2597,7 @@ class C1
 {
     static void Test(int arg1, (byte, byte) arg2)
     {
-        (int, int) t1 = new(int, int)();
+        (int, int)? t1 = new(int, int)?();
```

Comment:
Per current rules only a "new" applied immediately to a tuple type is not permitted. So this is valid. For now.

I am afraid generalizing the rule to a wider range of syntax is not so straightforward and unambiguous. I expect a range of opinions and valid reasons why we need to go one way or another from here, so I am not brave enough to guess :-)

If we want to cover a wider range of syntax, we need to tweak the design and implement it in another change.